app-frontend/src/app/core/services/featureFlag.directive.js

```diff
@@ -0,0 +1,59 @@
+export default (app) => {
```

Comment:
This would also be a good candidate to put in a short usage example so that the intended usage and any potential gotchas are clear.